Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Tower/FanSpeed/EnabledWSE/Subsequences/IntegratedOperation.mo

```diff
@@ -0,0 +1,299 @@
+                              conTyp==Buildings.Controls.OBC.CDL.Types.SimpleController.PID));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uChi[nChi]
+    "Chiller enabling status: true=ON"
+    annotation (Placement(transformation(extent={{-200,80},{-160,120}}),
+      iconTransformation(extent={{-140,60},{-100,100}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput chiLoa[nChi](
+    final unit=fill("W", nChi),
+    final quantity=fill("Power", nChi)) "Current load of each chiller"
+    annotation (Placement(transformation(extent={{-200,40},{-160,80}}),
+      iconTransformation(extent={{-140,-20},{-100,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uWSE
+    "Waterside economizer enabling status: true=ON"
+    annotation (Placement(transformation(extent={{-200,-20},{-160,20}}),
+      iconTransformation(extent={{-140,-100},{-100,-60}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yTowSpe(
+    final min=0,
+    final max=1,
+    final unit="1")
+    "Tower fan speed when WSE is enabled and there is any chiller running"
+    annotation (Placement(transformation(extent={{160,-100},{200,-60}}),
+      iconTransformation(extent={{100,-20},{140,20}})));
+
+protected
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Constant chiMinCycLoa[nChi](
+    final k=minUnLTon)
+    "Minimum cycling load of each chiller"
+    annotation (Placement(transformation(extent={{-120,130},{-100,150}})));
+  Buildings.Controls.OBC.CDL.Logical.Switch swi[nChi] "Logical switch"
+    annotation (Placement(transformation(extent={{-60,90},{-40,110}})));
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Constant zer[nChi](
+    final k=fill(0, nChi)) "Zero constant"
+    annotation (Placement(transformation(extent={{-120,70},{-100,90}})));
+  Buildings.Controls.OBC.CDL.Continuous.LimPID loaCon(
+    final controllerType=conTyp,
+    final k=k,
+    final Ti=Ti,
+    final Td=Td,
+    final yMax=1,
+    final yMin=0,
+    final reset=Buildings.Controls.OBC.CDL.Types.Reset.Parameter,
+    final y_reset=1)
+    "Controller to maintain chiller load at the sum of minimum cyclining load of operating chillers"
+    annotation (Placement(transformation(extent={{80,90},{100,110}})));
+  Buildings.Controls.OBC.CDL.Continuous.MultiSum totMinCycLoa(final nin=nChi)
+    "Sum of minimum cyclining load for the operating chillers"
+    annotation (Placement(transformation(extent={{-20,90},{0,110}})));
+  Buildings.Controls.OBC.CDL.Continuous.MultiSum totLoa(final nin=nChi) "Total load of operating chillers"
+    annotation (Placement(transformation(extent={{-20,50},{0,70}})));
+  Buildings.Controls.OBC.CDL.Continuous.MultiSum minCycLoa(final nin=nChi)
+    "Sum of minimum cyclining load for all chillers"
+    annotation (Placement(transformation(extent={{-20,130},{0,150}})));
+  Buildings.Controls.OBC.CDL.Continuous.Division div "Output first input divided by second input"
```

Comment:
I would rather implement this as it is described in 4.a). That means without the divisions. Also, in contrary of what the corresponding test shows, the fan speed should increase as the load increases above the minimum, in an attempt to increase the efficiency of the chiller by lowering the load.